doc/manpages/dinit-service.5.m4

```diff
@@ -360,6 +360,23 @@ This setting has no effect if the service is set to run on the console (via the
 \fBstarts\-on\-console\fR, or \fBshares\-console\fR options).
 The value is subject to variable substitution (see \fBVARIABLE SUBSTITUTION\fR).
 .TP
+\fBlogfile\-permissions\fR = \fIoctal-permissions-mask\fR
+Gives the permissions for the logfile specified using \fBlogfile\fR. Normally this will be 600 (user access
+only), 660 (user and group access), or 666 (all users).
+The default is 600.
+If logfile exist, thats perms will be ignored and the logfile permissions will be replaced with default or this option value.
+.TP
+\fBlogfile\-uid\fR = {\fInumeric-user-id\fR | \fIusername\fR}
+Specifies the user (name or numeric ID) that should own the logfile.
+If \fBlogfile\-uid\fR is specified as a name without also specifying \fBlogfile-gid\fR, then
+the logfile group is the primary group of the specified user (as found in the
+system user database, normally \fI/etc/passwd\fR).
+If the \fBlogfile\-uid\fR setting is not provided, the logfile will be owned by the user id of the \fBdinit\fR process.
+Also if logfile exist, thats uid will be ignored and the logfile will be owned by the user of \fBdinit\fR process or this option value.
```

Comment:
Replace lines 374-375 with:

"If the logfile already exists when the service starts, its ownership will be changed according to the value of this setting.
The default value is the user id of the \fBdinit\fR process."